.github/workflows/scripts/generate_multi_gcc_workflow.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import os
+from collections import OrderedDict
+import subprocess
+import re
+
+toolchain_dir = "/opt/arm"
+toolchains = os.listdir(toolchain_dir)
+
+gcc_versions = OrderedDict()
```

Comment:
Minor nitpick, but this doesn't need to be an OrderedDict, because `gcc_versions_sorted` gets constructed as an OrderedDict :wink:   (but of course leaving it as an OrderedDict doesn't hurt anything.)